test/msw-api/setup-server/scenarios/generator.node.test.ts

```diff
@@ -2,49 +2,61 @@ import fetch from 'node-fetch'
 import { rest } from 'msw'
 import { setupServer } from 'msw/node'
 
+type RequestParams = {
+  maxCount: string
+}
+
 const server = setupServer(
-  rest.get('/polling/:maxCount', function* (req, res, ctx) {
-    let count = 0
+  rest.get<never, RequestParams>(
+    '/polling/:maxCount',
+    function* (req, res, ctx) {
+      const maxCount = parseInt(req.params.maxCount)
```

Comment:
Request path parameters must be transformed to the expected type (i.e. string -> number).